frontend/src/styles/AppFooter.css

```diff
@@ -1,3 +1,16 @@
 .AppFooter {
```

Comment:
Borders between the icons are missing

Reply:
fixed by getting your help thank you very much.

Reply:
Resolved in dfbb89d080632919b27a6e5f383ce038e3d8958a